src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -190,6 +190,7 @@ public Button(ButtonStyle buttonStyle) : base(buttonStyle)
         protected override AccessibilityStates AccessibilityCalculateStates()
         {
             var states = base.AccessibilityCalculateStates();
+            states.Set(AccessibilityState.Checked, this.IsSelected);
```

Comment:
As "selected" state is supported in more general class (Button) it looks OK to move here the mapping to ATSPI for that state.

Reply:
Done.